src/main/java/teammates/logic/core/MailjetService.java

```diff
@@ -0,0 +1,62 @@
+package teammates.logic.core;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.jsoup.Jsoup;
+
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.resource.Email;
+
+import teammates.common.util.Config;
+import teammates.common.util.EmailWrapper;
+
+/**
+ * Email sender service provided by Mailjet.
+ * Reference: https://cloud.google.com/appengine/docs/java/mail/mailjet
+ * 
+ * @see MailjetClient
+ * @see MailjetRequest
+ * @see MailjetResponse
+ */
+public class MailjetService implements EmailSenderService {
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public MailjetRequest parseToEmail(EmailWrapper wrapper) {
+        MailjetRequest request = new MailjetRequest(Email.resource);
+        request.property(Email.FROMEMAIL, wrapper.getSenderEmail());
+        if (wrapper.getSenderName() != null && !wrapper.getSenderName().isEmpty()) {
+            request.property(Email.FROMNAME, wrapper.getSenderName());
+        }
+        
+        request.property(Email.RECIPIENTS, new JSONArray().put(new JSONObject().put("Email", wrapper.getRecipient())));
+        if (wrapper.getBcc() != null && !wrapper.getBcc().isEmpty()) {
+            request.property(Email.BCC, new JSONArray().put(new JSONObject().put("Email", wrapper.getBcc())));
```

Comment:
I'm not that certain, but according to http://dev.mailjet.com/guides/#sending-a-basic-email, at least for mailjet, we can't use bcc in conjunction with recipients?